### DISCUSSION TITLE
Downloading Multiple Language Subtitles

### DISCUSSION THREAD

**Discussion by kenizl86** [pymedusa/Medusa#9276] [General]
Hello, I hope I can post this general question here. I tried researching what I could on this, but nothing has come up.

My issue is very simple, however I believe my problem can include others using Medusa for the same purposes. I am learning Japanese and am using Anime as a primary input method. Thus, I would like to have the episodes come with Japanese subtitles, but also with English (for various reasons). I added both to the "Subtitle Languages" under "Subtitle Settings" but so far out of 17 shows I've added, it only pulls English.

Now I have a feeling this isn't a problem with Medusa but rather my search providers, but I would just like to confirm that Medusa will in fact attempt to search for both subtitles and not just English.

In this same vein, is there a way to add other subtitle search providers to the list? For example, Kitsunekko.net has primarily Jap subtitles and would be fantastic for this.

In case it is relevant, I am not using Usenet (because I could not figure out how to get it running) but am instead using Torrents, with my main searches being in Nyaa, TokyoToshokan and AniDex. I've enabled Addic7ed (and pretty much every other plugin) and was going to use OpenSubtitles but couldn't find any Jap subs there.

Thanks!

**Comment by kenizl86** [comment]
I actually figured out it does have to do with the providers I am using, as now it is (at least attempting and sometimes successfully) downloading Japanese and English subs.

However, my other question in this same line stands: is there a way to add other subtitle search providers to the list? For example, Kitsunekko.net and others like it.

**Reply by medariox** [reply]
Medusa uses subliminal under the hood for the subs download, so you will have to ask them: https://github.com/Diaoul/subliminal
We will add it whenever they do.
